### DISCUSSION TITLE
How-To use ForceTLS

### DISCUSSION THREAD

**Discussion by Abraham1809** [inbucket/inbucket#512] [General]
Could you provide some insights how to use ForceTLS

Thank you

**Comment by Abraham1809** [comment]
Actually I only asked, which env variable should be set in order to activate ForceTLS.

**Reply by jhillyerd** [reply]
It should be `INBUCKET_SMTP_FORCETLS` = true

It was a feature contributed to the project, seems like it didn't make it into the configuration doc.

https://github.com/inbucket/book.inbucket.org/issues/17
